src/backend/dx11/Cargo.toml

```diff
@@ -27,7 +27,7 @@ bitflags = "1"
 libloading = "0.6"
 log = { version = "0.4" }
 smallvec = "1.0"
-spirv_cross = { version = "0.22", features = ["hlsl"] }
+spirv_cross = { version = "0.22.2", features = ["hlsl"] }
```

Comment:
why are we changing the version everywhere? we only require `0.22.2` in the GL backend, no need to touch the other manifests

Reply:
Sorry, misunderstood you. Fixed.